test/history_test.js

```diff
@@ -182,13 +182,13 @@ describe('History', function() {
           await browser.visit('/');
           browser.history.pushState({ is: 'first' }, null, '/first');
           browser.history.pushState({ is: 'second' },   null, '/second');
-          browser.back();
           browser.window.addEventListener('popstate', function(event) {
             lastEvent = event;
-            done();
           });
+          browser.back();
           browser.history.forward();
           browser.wait().done();
+          done();
```

Comment:
Before this change: call `browser.wait` and wait until the event gets triggered, then proceed to next step.

After this change: call `browser.wait` and proceed to next step. The thing, `browser.wait` is asynchronous, doesn't guarantee any work will be done before `browser.wait()` returns (only before the promise resolves).